### PR TITLE
feat(subscriber history): add icons buttons and validations

### PR DESCRIPTION
**Pull request by GusBaamonde**
#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

form-subscribers-history

### CODE REVIEW

**Comment by cbernat** [comment]
One question, right now we have only the first input, this does scale right? I mean, the search input streatchs to fill all width when the other three fields are not present?

**Comment by GusBaamonde** [comment]
> Una pregunta, en este momento solo tenemos la primera entrada, ¿esto escala bien? Quiero decir, ¿la búsqueda ingresa streatchs para llenar todo el ancho cuando los otros tres campos no están presentes?

If perfect scale because we use the grid system to position the fields we need

**Comment by andresmoschini** [comment]
👍

**Comment by cbernat** [comment]
Very nice! ❤️

**Comment by GusBaamonde** [comment]
SH-accesibilidad-01

File: assets/scss/modules/_images.scss [hunk 1]
```diff
@@ -23,54 +22,63 @@
...
 // User removed soft bounces
-.icon-user--rsb:before {
+.user-removed-soft-bounces:before {
   @include icon-user-table;
   background-image: url('../../img/user-removed-soft-bounces.svg');
 }
 
 // User subscriber removed
-.icon-user--sr:before {
+.user-subscriber-removed:before {
```
Comment (cbernat) [inline assets/scss/modules/_images.scss]: How about changing the order here to follow consistency? like `removed-subscriber`

File: assets/scss/modules/_images.scss [hunk 2]
```diff
@@ -23,54 +22,63 @@
...
 // User active
-.icon-user--active:before {
+.user-active:before {
   @include icon-user-table;
   background-image: url('../../img/user-active.svg');
 }
 
 // User active not associated to lists
-.icon-user--natl:before {
+.user-active-with-no-list:before {
```
Comment (cbernat) [inline assets/scss/modules/_images.scss]: Are we not using BEM here for class naming? I think here perhaps `user--active-with-no-list` would be best to represent element with modifier. What do you think?

**Comment by GusBaamonde** [comment]
cambio-clases

**Pull request merged**